find-plugin-repos.sh

```
@@ -114,6 +115,7 @@
   # Create a CSV file and write the header
   echo "Plugin,URL" >"$csv_file"
   echo "Plugin,URL" >"$csv_file_no_jenkinsfile"
+  echo "Plugin,URL" >"$csv_file_jdk11"
```

Shellcheck (0.10.0)

[warning] csv_file_jdk11 is referenced but not assigned.

(SC2154)